extensions/wikia/AdEngine/js/lookup/prebid/adapters/fastlane.js

```diff
@@ -0,0 +1,162 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.fastlane', [
```

Comment:
Did we consider reusing existing `rubicon` adapter instead of creating new one? What are benefits of using separate adapters?